opencog/guile/CMakeLists.txt

```diff
@@ -36,6 +37,7 @@ TARGET_LINK_LIBRARIES(logger
 	smob
 	${GUILE_LIBRARIES}
 	${COGUTIL_LIBRARY}
+    ruleengine # for the ure logger
```

Comment:
I guess I'm suggesting something like this: editing `opencog.scm` and adding a line like
```
(load-extension "libure" "opencog_ure_init")
```
or something like that. I'd have to dive into the guile documentation to figure this out, precisely.  I'm sort of skimming your code, and I'm guessing that it the only thing you want is the logger, that it would be enough to load the shared library, like this, and you'd get it ... !?  Maybe there's a gotcha. Dunno.